### DISCUSSION TITLE
Jul 1 pricing

### DISCUSSION THREAD

**Discussion by rob0101** [amberelectric/public-api#43] [Q&A]
Has the API been updated for SA’s new pricing for Jul 1 ?

Ta.

**Comment by madpilot** [comment]
I wasn't in the office today, but the tariff definitions were uploaded last night, so today's prices should reflect the new prices - are you seeing a discrepancy, or just asking?

**Reply by rob0101** [reply]
just asking before I look at Jul 1 usage tomorrow.
